package.json

```diff
@@ -39,13 +39,15 @@
     "typescript": "^3.3.3"
   },
   "dependencies": {
+    "applicationinsights": "^1.0.3",
```

Comment:
are these new versions needed for this PR? it doesn't seem so

Reply:
It is in sync with the one in commons